specification/2.0/README.md

```diff
@@ -2183,38 +2150,64 @@ Application-specific data.
 <a name="reference-material"></a>
 ## material
 
-The material appearance of a primitive.
-
-**Properties**
-
-|   |Type|Description|Required|
-|---|----|-----------|--------|
-|**technique**|`string`|The ID of the [`technique`](#reference-technique).|No|
-|**values**|`object`|A dictionary object of parameter values.|No, default: `{}`|
-|**name**|`string`|The user-defined name of this object.|No|
-|**extensions**|`object`|Dictionary object with extension-specific objects.|No|
-|**extras**|`any`|Application-specific data.|No|
-
-Additional properties are not allowed.
-
+glTF 2.0 uses the metal-roughness material model from Physically Based Renderring. The metal-roughness material model is defined by the following properties:
+* Base Color: The base color of the material
+* Metallic-ness: A value between 0 and 1, indicating if the material is metallic (1) or not (0)
```

Comment:
The [Sketchfab models](https://sketchfab.com/features/gltf) include metallic-ness values that are between 0 and 1, not just 0 or 1. Is this an exception to the spec?